app/views/static_pages/about/_overview.html.erb

```diff
@@ -1,51 +1,54 @@
-<section class="overview odin-dark-bg">
+<section class="py-12 sm:py-24 bg-gray-100">
   <div class="container">
-    <h2 class="overview__heading accent text-2xl">Overview of The Odin Project</h2>
+    <h2 class="mb-12 text-gold text-center text-2xl font-bold">Overview of The Odin Project</h2>
 
-    <div class="row">
-
-      <div class="col-12 col-md-4 col-lg-4">
-        <div class="odin-stat card-main odin-dark-bg-accent">
-          <p class="odin-stat__content"><%= User.count %></p>
+    <div class="grid grid-cols-1 sm:grid-cols-3 gap-x-8 gap-y-16">
+      <%= render CardComponent.new do |card| %>
```

Comment:
[BUG] The stats cards aren't going full width at below the sm breakpoint:
<img width="480" alt="Screenshot 2022-07-18 at 00 59 14" src="https://user-images.githubusercontent.com/7963776/179430130-cecd00de-69b1-4e26-bd15-aa5667d3d7cb.png">